### PR TITLE
Add new column_widgets to create_repos_table

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Signed-off-by: Gaurav Talreja <gtalreja@redhat.com>

### CODE REVIEW

File: airgun/entities/user.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+from wait_for import wait_for
 
 from airgun.entities.base import BaseEntity
 from airgun.navigation import NavigateStep
 from airgun.navigation import navigator
 from airgun.views.user import UserCreateView
 from airgun.views.user import UserDetailsView
 from airgun.views.user import UsersView
-from wait_for import wait_for
+
```
Comment (omkarkhatavkar) [inline airgun/entities/user.py:10]: remove the extra space

**Review by omkarkhatavkar — APPROVED** [review]
ACK, Provide the Test Result.

**Review by mirekdlugosz — APPROVED** [review]
Changes look sane, but by convention we do require demonstration that proposed change do work. It might be link to robottelo PR with test results, if these changes are needed for test you are writing.

**Review by lhellebr — APPROVED** [review]
Looks good. Where is this used? Are there any related entity changes?

File: airgun/entities/user.py (hunk 1 above)
Comment (omkarkhatavkar) [reply airgun/entities/user.py:10]: spoke Gaurav this is specific to precommit

Comment (Gauravtalreja1) [reply airgun/entities/user.py:10]: Thanks, Omkar for putting it here, all changes to this file `airgun/entities/user.py` are due to pre-commit

**Comment by omkarkhatavkar** [comment]
> Looks good. Where is this used? Are there any related entity changes?

@lhellebr https://github.com/SatelliteQE/robottelo/pull/8021 is the PR consuming

**Pull request merged**